src/Sylius/Bundle/AdminBundle/Resources/views/Promotion/Field/usedWithUsageLimit.html.twig

```diff
@@ -0,0 +1,11 @@
+<div class="ui red label">
+    {{ data.used }}
+    <div class="detail">
+        /
+        {% if data.usageLimit %}
+                {{ data.usageLimit }}
+        {% else %}
+                ∞
```

Comment:
Then we could avoid this ``else`` which is quite strange - I think it's obvious that promotion has no usage limit if it's not set :)